ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -73,12 +73,13 @@
 
         setModel(model) {
             super.setModel(model);
-            let widgets = this.widget
+            let widgets = this.widget;
             widgets.forEach(widget => {
-                let self = widget
+                let self = widget;
+                this._updateModelValue(self);
```

Comment:
this will lead to invocation of subscribe as well. Though it would not lead to cycle, but can we somehow prevent that extra call as well ?

Reply:
no, it will not. subscribe is not inside updateModelValue.